src/cascadia/TerminalSettingsModel/Microsoft.Terminal.Settings.ModelLib.vcxproj

```diff
@@ -327,7 +327,7 @@
 
   <!-- PowerShell version check, outputs error if the wrong version is installed. -->
   <Target Name="_WindowsPowershellVersionCheck" BeforeTargets="BeforeClCompile">
-    <Exec Command="powershell.exe -NoProfile -ExecutionPolicy Unrestricted &quot;$(OpenConsoleDir)\tools\WindowsCheckPSVersion.ps1&quot;" />
+    <Exec Command="powershell.exe -NoProfile -ExecutionPolicy Unrestricted &quot;&amp;&apos;$(OpenConsoleDir)\tools\WindowsCheckPSVersion.ps1&apos;&quot;" />

```

Comment:
```
"&'$(OpenConsoleDir)\toolsWindowsCheckPSVersion.ps1'"
```
...that doesn't seem right. 🤔

Reply:
I find it quite strange too, I found out you need to use ampersand which can be only used inside double quot after surfing a bit on [stackoverflow](https://stackoverflow.com/questions/45760457/how-can-i-run-a-powershell-script-with-white-spaces-in-the-path-from-the-command). After some testing in command line I found this version as the solution.
![image](https://github.com/microsoft/terminal/assets/47116521/28a41a31-ce57-428b-a9f7-04571bb1b3db)
![image](https://github.com/microsoft/terminal/assets/47116521/8ec85086-93fe-4d82-bc87-c71d68245bea)
Just to ensure the script is indeed running, because it gave me 0 response I removed the -not from the if inside the script and tried running this way so in this setup I get an error when I have the pwsh.exe installed correctly.
![image](https://github.com/microsoft/terminal/assets/47116521/b81cbd96-a43d-4286-8c6e-810beac8a2eb)
![image](https://github.com/microsoft/terminal/assets/47116521/44a3c806-5dc6-4b03-839d-b1e2d821945d)
Then I built the project successfully using VS inside a folder with space inside its name. If there are anything I didn't consider while testing this solution feel free to share with me.
![image](https://github.com/microsoft/terminal/assets/47116521/56e5dab3-28da-42bb-bae2-2d7a02028864)

Reply:
The StackOverflow link implies that we can just use `-File`. Does that work as well?

Reply:
If you don't mind, it'd be fantastic if you could make that change. It would sort of serve as a guide how to do it "right" to future developers, whereas the & trick seems a bit like a hack to be entirely honest. 😅

Reply:
> If you don't mind, it'd be fantastic if you could make that change. It would sort of serve as a guide how to do it "right" to future developers, whereas the & trick seems a bit like a hack to be entirely honest. 😅

Okay, I modify the solution using this format and if the build still successfull I commit my changes.

Reply:
After testing I decided to postpone the fix a bit, because after testing I found other projects with the same problem.
Before
![image](https://github.com/microsoft/terminal/assets/47116521/87cbf4c5-87d7-4030-be18-deee91a4e45c)
![image](https://github.com/microsoft/terminal/assets/47116521/52769ab4-c019-4582-988b-4bce51631f58)
After
![image](https://github.com/microsoft/terminal/assets/47116521/c1d3559e-4b15-4beb-bd84-a3d77ea59294)
![image](https://github.com/microsoft/terminal/assets/47116521/89894103-1183-4dbe-8771-971463f86b79)